next.config.js

```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  images: {
+    domains: ['i.pinimg.com'],
+  },
```

Comment:
❓ 
I didn't get why did you do that?

Reply:
Ok, let me tell you what happen to me when I worked on images this error "Error: Invalid src prop [(https://i.pinimg.com/236x/da/c5/f8/dac5f861cf4a5d0cfbe8f310016bea28.jpg)](https://i.pinimg.com/236x/da/c5/f8/dac5f861cf4a5d0cfbe8f310016bea28.jpg) on `next/image`, hostname "i.pinimg.com" is not configured under images in your `next.config.js`
 This happened because I didn't add the domains of these images in the future we can replace this domain with the domains of users' images coming from the database. this is what I got from my search on this error.